docs/streaming/apis-on-dataframes-and-datasets.md

```diff
@@ -18,6 +18,7 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+{% include rel_path_to_root.html %}
```

Comment:
Note for reviewer: you can't access Jekyll variables even if they are defined in the layout you are using. I couldn't find official docs for this, but there's consensus on SO: https://stackoverflow.com/questions/30011684/can-you-use-jekyll-layout-variables-in-pages